webapp/packages/core-connections/src/locales/zh.ts

```diff
@@ -44,12 +44,6 @@ export default [
   ['connections_connection_create_fail', '无法创建连接'],
   ['connections_connection_save_fail', '无法保存连接'],
   ['connections_connection_expert_settings', '专业设置'],
-  ['connections_connection_keep_alive', '保持连接 (秒)'],
-  ['connections_connection_autocommit', '自动提交'],
-  ['connections_connection_default_catalog', 'Default catalog'],
-  ['connections_connection_default_schema', 'Default schema'],
-  ['connections_connection_keep_alive_tooltip', '无自动断开连接'],
-  ['connections_connection_read_only', 'Read-only connection'],
```

Comment:
We need to keep localizations if they were not changed, but just moved to the server side